dockerfile/manylinux_2_28.Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM almalinux:8 AS buildenv
+
+RUN dnf install -y gcc-toolset-13 python3.11 python3.11-devel \
+    zstd libzstd-devel xz-devel zlib-devel git which vim && \
+    update-alternatives --set python /usr/bin/python3.11 && \
+    update-alternatives --set python3 /usr/bin/python3.11 && \
+    python -m ensurepip && \
+    python -m pip install cmake ninja wheel
+RUN mkdir /root/build
+
+ARG amdgpu_installer="https://repo.radeon.com/amdgpu-install/6.2.2/el/8.10/amdgpu-install-6.2.60202-1.el8.noarch.rpm"
```

Comment:
This is a maintenance headache, we need to make this more generic for this to be generally usable for any ROCm release

Reply:
Discussed this offline, this is the default value. The intention is for any CI job or users to provide a link to the desired amdgpu installer for their desired ROCm version.
We could use logic similar to this file:
https://raw.githubusercontent.com/wiki/ROCmSoftwarePlatform/pytorch/files/install_kdb_files_for_pytorch_wheels.sh
to keep the default argument always up-to-date.